packages/@o3r/dev-tools/src/cli/set-version.ts

```diff
@@ -6,6 +6,7 @@ import * as fs from 'node:fs';
 import * as path from 'node:path';
```

Comment:
Maybe we could align on what is done on https://github.com/AmadeusITGroup/otter/pull/868 and use `spawnSync` to avoid keep duplicated code

Reply:
I would say no for the same justification given here:
https://github.com/AmadeusITGroup/otter/pull/766#discussion_r1348170445